pipeline/metadata/flatten_satellite.py

```diff
@@ -371,54 +464,60 @@ def _process_satellite_v2_control(
         Rows
     """
     responses = scan.get('response', [])
```

Comment:
Please rename `scan` to `responses_entry`. Introduce a `ResponsesEntry` type that corresponds to the responses.json and responses_control.json files (as defined at https://docs.censoredplanet.org/dns.html)

Reply:
Also specify in the comment the filename this is coming from.

Reply:
done and done